config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root "home#index"
```

Comment:
- [ ] This error message appears when I started localhost:

![error](https://user-images.githubusercontent.com/4246637/73844717-c689cc80-4829-11ea-9310-b536b0be9d66.png)

You are setting the root to index action of home controller which doesn't exist. -- Let's fix this.